docs/changelogs/CHANGELOG-2.21.md

```diff
@@ -15,6 +15,15 @@
 - [v2.21.12](#v22112)
 - [v2.21.13](#v22113)
 - [v2.21.14](#v22114)
+- [v2.21.15](#v22115)
+
+## [v2.21.15](https://github.com/kubermatic/kubermatic/releases/tag/v2.21.15)
+
+With this release, support for KKP v2.21.x ceases. Please upgrade to a supported version of KKP in the near future.
```

Comment:
[This bird is no more. It has ceased to be!](https://www.youtube.com/watch?v=vZw35VUBdzo)